examples/redis-demo/src/test/groovy/com/example/ProxyAwareSpec.groovy

```diff
@@ -1,13 +1,10 @@
 package com.example
 
-import grails.core.support.proxy.ProxyHandler
-
 trait ProxyAwareSpec {
 
-    ProxyHandler proxyHandler
```

Comment:
THis cause an NPT since the proxyHandler is not wired up, setting in the one test fixes this issue.